src/test/resources/emails/sessionReminderEmailCopyToInstructor.html

```diff
@@ -0,0 +1,87 @@
+<p>Hello Instructor1 Course1,</p>
+
+<p>The email below has been sent to students of course: [idOfTypicalCourse1] Typical Course 1 with 2 Evals.<br>
+<br>
+=== Email message as seen by the students ===</p>
+
+
+<p>
+  The following feedback session is still open for submissions.
+  <div>
+    <table style="max-width:600px;border:1px solid black;">
+      <tr>
+        <td style="padding:5px;">
+          <strong>
+            Course:
+          </strong>
+        </td>
+        <td style="padding:5px;">
+          [idOfTypicalCourse1] Typical Course 1 with 2 Evals
+        </td>
+      </tr>
+
+      <tr>
+        <td style="padding:5px;">
+          <strong>
+            Feedback Session Name:
+          </strong>
+        </td>
+        <td style="padding:5px;">
+          First feedback session
+        </td>
+      </tr>
+
+      <tr>
+        <td style="padding:5px;">
+          <strong>
+            Deadline:
+          </strong>
+        </td>
+        <td style="padding:5px;">
+          Fri, 30 Apr 2027, 11:59 PM UTC+0200
```

Comment:
The format has changed after #84.

Reply:
Oh! Thank you for pointing out. I was wondering why the build failures, and as I am currently working on the other PR, didn't run the tests. Will change it accordingly.